src/app/newSettings/Sections/General/components/Appearance.tsx

```diff
@@ -55,13 +55,25 @@ const Appearance = () => {
 
   // CHECK IF THIS WORKS BEFORE MERGE
   useEffect(() => {
+    function isAppearanceExist(prevAppearances, theme, img) {
+      return prevAppearances.some((appearance) => appearance.theme === theme && appearance.img === img);
+    }
+
+    function addAppearanceIfUnique(prevAppearances, theme, img) {
+      if (isAppearanceExist(prevAppearances, theme, img)) {
+        return prevAppearances;
+      }
+      return [...prevAppearances, { theme, img }];
+    }
+
+    const updateAppearance = (theme, img) => {
+      setAppearances((prevAppearances) => addAppearanceIfUnique(prevAppearances, theme, img));
+    };
```

Comment:
Maintain consistency in the functions, here some functions are in arrow type and the others in regular style.
The type of the arguments and the return of the functions are missing.